src/software/ai/hl/stp/tactic/stop_tactic.cpp

```diff
@@ -6,9 +6,7 @@
 #include "software/ai/hl/stp/tactic/mutable_tactic_visitor.h"
 
 
-StopTactic::StopTactic(bool coast, bool loop_forever) : Tactic(loop_forever), coast(coast)
-{
-}
+StopTactic::StopTactic(bool coast) : Tactic(true), coast(coast) {}
```

Comment:
The boolean is for `coast` rather than 'loop_forever'. `StopTactic` is not going to expose `loop_forever` to user

Reply:
Ah didn't realize that. This should be fine then.